railties/lib/rails/configuration.rb

```diff
@@ -65,6 +74,19 @@ def use(*args, &block)
       def delete(*args, &block)
         @delete_operations << -> middleware { middleware.send(__method__, *args, &block) }
       end
+      ruby2_keywords(:delete) if respond_to?(:ruby2_keywords, true)
```

Comment:
These three `ruby2_keywords` are not necessary because `delete`, `move_before`, and `move_after` doesn't take keyword arguments.

Reply:
Thanks, I've removed the `ruby2_keywords` from the Action Pack code yesterday, but did not take the proxy into account. I have removed them from here as well!